app/src/test/java/io/intrepid/contest/screens/contestcreation/describecontest/DescribeContestPresenterTest.java

```diff
@@ -15,20 +16,29 @@
 public class DescribeContestPresenterTest {
     @Mock
     DescribeContestContract.View mockView;
-    private DescribeContestContract.Presenter describeContestPresenter;
+    @Mock
```

Comment:
So we can actually get the full code coverage here, we just need to use PowerMock instead of regular mockito. Give it a go and message me on slack if you get stuck. PowerMock can be a bit harder to use than regular mockito but should be usable. You just need to remember to switch the test runner and also add an @preparefortest annotation.